t/ui/13-admin.t

```diff
@@ -563,13 +563,41 @@ subtest 'edit the yaml' => sub() {
     $driver->find_element_by_id('preview-template')->click();
     my $result = $form->child('.result');
     wait_for_ajax;
-    ok($result->get_text() =~ m/Preview of the YAML/, 'preview shown') or diag explain $result->get_text();
+    ok($result->get_text() =~ m/Preview of the changes/, 'preview shown') or diag explain $result->get_text();
```

Comment:
Note that you could also use `like()` from `Test::More` for all these regex checks.